.travis.yml

```diff
@@ -19,3 +19,11 @@ matrix:
     - rvm: jruby-head
     - rvm: rbx-3
   fast_finish: true
+script:
+  # Unit test
+  - bundle exec rake
+  # Install test
+  - gem build rb-inotify.gemspec
```

Comment:
This test is to prove `rb-inotify.gemspec` itself is valid such as runtime and development dependency. Sometimes this command becomes error or displays warning message if it is not good.